protocol/x/affiliates/keeper/msg_server.go

```diff
@@ -38,25 +37,8 @@ func (k msgServer) UpdateAffiliateTiers(ctx context.Context,
 	}
 
 	sdkCtx := sdk.UnwrapSDKContext(ctx)
-	unconditionalRevShareConfig, err := k.revShareKeeper.GetUnconditionalRevShareConfigParams(sdkCtx)
```

Comment:
Not needed anymore since we check only the max affiliate rev share cap

Reply:
Do we still need a reference to `revShareKeeper` on `affiliateKeeper`?